app/controllers/katello/concerns/api/v2/content_overrides_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Katello
+  module Concerns
+    module Api::V2::ContentOverridesController
+      extend ActiveSupport::Concern
+
+      # overriding_object => pass it either an activation key or content host
+      def validate_content_overrides_enabled(content_params, overriden_object)
+        valid_values = ["yes", "no", "0", "1", "default"]
+        unless valid_values.include?(content_params[:value].to_s)
+          fail HttpErrors::BadRequest, _("Value must be one of 0/1, or yes/no, or 'default'")
+        end
+
+        unless overriden_object.valid_content_override_label?(content_params[:content_label])
+          fail HttpErrors::BadRequest, _("Invalid content label: %s") % content_params[:content_label]
+        end
+
+        override = ::Katello::ContentOverride.new(content_params[:content_label])
+        case content_params[:value].to_s.downcase
+        when 'default'
+          override.enabled = nil
+        when '1', "yes"
+          override.enabled = '1'
+        when '0', "no"
+          override.enabled = '0'
+        end
```

Comment:
Why did you move this logic out of the ContentOverride class? Given how specific it is to a content override, this seemed like good information to store there and retrieve from the instance of the class by passing in the `content_params[:value]`.